frontend/src/atoms.ts

```diff
@@ -234,6 +218,20 @@ export function useAppOCPSearchResultLimit() {
   return useMemo(() => parseInt(settings.APP_OCP_SEARCH_RESULT_LIMIT ?? '1000'), [settings])
 }
 
+export function useVitualMachineActionsEnabled() {
```

Comment:
Do you want to just fix this typo before merging?

Reply:
Yes, thank you!